docs/docs/gatsby-style-guide.md

```diff
@@ -131,79 +135,52 @@ Note: Just to clarify, you can include special characters in the article title
 but _not_ in the folder name (e.g. Title: What is GraphQL? and Folder Name:
 what-is-graphql).
 
-# Modularity
+### Introductory paragraph
 
-Each article should explain exactly one concept, and that concept should be
-apparent from the article's title.
+The introduction paragraph should be a 1-2 sentence explanation of the main
```

Comment:
The title and first line use _introductory_ and _introduction_. Should it be the same word in both places?